loopabull/main.py

```diff
@@ -64,11 +62,10 @@ def load_config(self, config_path):
             sys.exit(1)
 
         try:
-            self.ansible = config["ansible"]
-            if 'inventory_path' not in self.ansible:
-                raise IndexError
-            if 'playbooks_dir' not in self.ansible:
-                raise IndexError
+            self.ansible = config['ansible']
+            self.ansible['command']
+            self.ansible['playbook_dir']
```

Comment:
I think this is just a typo of `playbook_dir` vs `playbooks_dir` (I'm open to changing the name though if you think one is better than the other).